calblueprint/siren-web
# Andrei/buttons intake

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
Implementing buttons for intake form (add, delete, undo-redo).
## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [ x] Re-implement Add to useReducer
 - [ x] Implement delete button
 - [] Implement undo-redo

## How can the reviewer test your code? 👩‍💻
_Use add button on overlay and click trash icon next to every question.  Undo and redo are on overlay.  Check if text in questions are preserved during undo-redo_ 

## Any bugs you encountered or still having trouble with? 🐛
_Undo-redo still janky.  When redoing an undo, the restored question is reset.  Also going over getting history correct.  Need a better way to store history.  I think it has to do with using shallow copies to store questions._ 


## Resources 📔
_link online resources and related PRs_ 


🧜‍♀️ cc: @shannonbonet @phoebeli23
